client/views/omnichannel/currentChats/FilterByText.tsx

```diff
@@ -45,7 +47,8 @@ const FilterByText: FilterByTextType = ({ setFilter, reload, ...props }) => {
 	const [from, setFrom] = useLocalStorage('from', '');
 	const [to, setTo] = useLocalStorage('to', '');
 	const [tags, setTags] = useLocalStorage<never | { label: string; value: string }[]>('tags', []);
-	const [customFields, setCustomFields] = useLocalStorage<any[]>('tags', []);
+	const [customFields, setCustomFields] = useLocalStorage<{}>('customFields', {});
```

Comment:
I'm not sure, I went with this approach since I wanted to make sure that the state is preserved when the customer moves away from this component (like if they open an ongoing chat), and then come back. If there is some other approach to handle this, do let me know